pubsub/rabbitmq/rabbitmq.go

```diff
@@ -362,6 +357,11 @@ func (r *rabbitMQ) subscribeForever(req pubsub.SubscribeRequest, queueName strin
 				break
 			}
 
+			if !subscribed {
+				subscribed = true
+				ackCh <- true
```

Comment:
Super small nit: I believe you want to set `ackCh` to nil so that it gets garbage collected.